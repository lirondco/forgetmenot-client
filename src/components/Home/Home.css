
.home_component {
    padding: 5%;
    display: flex;
    flex-direction: column;
    align-items: center;
    padding-bottom: 100%;
}


.signin, .signup {
    background: #C03D72;
    color: #EFEBE9;
    margin: 20% 0 20% 0;
    width: 120px;
    height: 40px;
    text-align: center;
    font-size: 18pt;
    border-radius: 20%;
}

.home_component a {
    text-decoration: none;
    font-family: 'Patrick Hand SC', cursive;
}