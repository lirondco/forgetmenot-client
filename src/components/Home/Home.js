import React from 'react';
import { Link } from 'react-router-dom';
import './Home.css'

export default class Home extends React.Component {
    render() {
        return (
            <div className ='home_component'>
                <Link to="/login">
                    <div className="signin">
                        Log In
                    </div>
                </Link>
                <Link to="/join">
                    <div className="signup">
                        Sign Up
                    </div>
                </Link>
            </div>
        )
    }
}