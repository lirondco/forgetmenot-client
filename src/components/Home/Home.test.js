import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Home from './Home'

describe(`Home component`, () => {
  it('renders the home page', () => {
    const wrapper = shallow(<Home />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})