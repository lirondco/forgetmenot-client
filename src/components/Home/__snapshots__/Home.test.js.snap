// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders the home page 1`] = `
<div
  className="home_component"
>
  <Link
    to="/login"
  >
    <div
      className="signin"
    >
      Log In
    </div>
  </Link>
  <Link
    to="/join"
  >
    <div
      className="signup"
    >
      Sign Up
    </div>
  </Link>
</div>
`;
